base_field_validator/models/ir_model_field_regex.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    See __openerp__.py about license
+#
+##############################################################################
```

Comment:
Some of us would thank you if you use the new-style license headers found in the official template. It's more accurate and less dramatic to the eye.